apps/src/applab/mode-javascript_codeorg.js

```diff
@@ -0,0 +1,51 @@
+// define ourselves for ace, so that it knows where to get us
+ace.define("ace/mode/javascript_codeorg",["require","exports","module","ace/lib/oop","ace/mode/javascript","ace/mode/javascript_highlight_rules","ace/worker/worker_client","ace/mode/matching_brace_outdent","ace/mode/behaviour/cstyle","ace/mode/folding/cstyle","ace/config","ace/lib/net"], function(acerequire, exports, module) {
```

Comment:
Line is too long.<br>'ace' is not defined.<br>'module' is defined but never used.